### DISCUSSION TITLE
Pressure plate collision

### DISCUSSION THREAD

**Discussion by TheLeCrafter** [Minestom/Minestom#1308] [Help] [answered]
Hello, I am trying to create working pressure plates. I use the PlayerMoveEvent to check if the block's namespace id of the new location contains "pressure_plate". I then use the following code to check if the player's bounding box and the block are colliding:
`newBlock.registry().collisionShape().intersectBox(event.newPosition.sub(event.player.position.roundToBlock()), event.player.boundingBox)`. This code somehow always returns false and I don't know why. I saw [this](https://github.com/Minestom/Minestom/discussions/1248) post, but I thought that there must be a solution that doesn't require this hacky workaround.

**Comment by MrlnHi** [comment, marked as answer]
pressure plates don't have a collision box. you can walk through them

**Reply by TheLeCrafter** [reply]
Okay so there is no other way to check if the player stands on the pressure plate?

**Reply by MrlnHi** [reply]
well I would say check if any corner of their hitbox is in the block? idk exactly how mc vanilla does it
